### DISCUSSION TITLE
Theming

### DISCUSSION THREAD

**Discussion by DavidRobertKeller** [epoupon/fileshelter#49] [General]
Hello,
I take a look at your demo site here https://fileshelter.demo.poupon.io/
I see a repo called 'theme' on your website : https://fileshelter.demo.poupon.io/resources/themes/bootstrap/3/less/jumbotron.less
How can I get the Theme Files ?
How can I implement it on my own instance ?
Thanks a lot and best regards
David

**Comment by epoupon** [comment]
Hi!
Sorry for late response, just getting back to dev on it.
There is easy theming at the moment, I will consider adding this for the v5 branch
